README.md

```diff
@@ -71,18 +75,42 @@ optional arguments:
                         This config will be used for the tmux session
   -d, --detach          Start session in detached mode
   --overwrite OVERWRITE
-                        Overwrite a parameter from the session config.
-                        Parameters can be specified using a comma-separated
-                        list such as '--overwrite param_a=1,param_b=2'.
+                        Overwrite a parameter from the session config. Parameters can be specified
+                        using a comma-separated list such as '--overwrite param_a=1,param_b=2'.
 ```
 
 ### Full blown example
-To make use of all catmux features, run the following example command:
+To make use of all catmux features, run the following example command, again from the root
+directory:
 ```
-rosrun catmux create_session package://catmux/etc/example_session.yaml --tmux_config package://catmux/etc/tmux_default.conf --session_name example_session --overwrite show_layouts=True,replacement_param="new catmux user"
+catmux_create_session ${HOME}/.local/share/catmux/example_session.yaml \
+  --tmux_config {HOME}/.local/share/catmux/tmux_default.conf \
+  --session_name example_session \
+  --overwrite show_layouts=True,replacement_param="new catmux user"
```

Comment:
```suggestion
# In case you used the `--user` flag during the install process
catmux_create_session ${HOME}/.local/share/catmux/example_session.yaml \
  --tmux_config {HOME}/.local/share/catmux/tmux_default.conf \
  --session_name example_session \
  --overwrite show_layouts=True,replacement_param="new catmux user"
  
  
# In case you used `sudo pip install ...` for installation
catmux_create_session $(python3 -c "import sys; print(sys.prefix)")/share/catmux/example_session.yaml \
  --tmux_config $(python3 -c "import sys; print(sys.prefix)")/share/catmux/tmux_default.conf \
  --session_name example_session \
  --overwrite show_layouts=True,replacement_param="new catmux user"
```

Reply:
Using `--user` the full-blown example from the README works.